com.sap.cloud.lm.sl.cf.database/src/main/java/com/sap/cloud/lm/sl/cf/database/migration/generator/DatabaseTableInsertQueryGenerator.java

```diff
@@ -0,0 +1,48 @@
+package com.sap.cloud.lm.sl.cf.database.migration.generator;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+import com.sap.cloud.lm.sl.cf.database.migration.metadata.DatabaseTableColumnMetadata;
+import com.sap.cloud.lm.sl.cf.database.migration.metadata.DatabaseTableMetadata;
+
+public class DatabaseTableInsertQueryGenerator {
+
+    private static final String CLOSING_BRACKET = ")";
+    private static final String OPEN_BRACKET = "(";
+    private static final String DEFAULT_STATEMENT_VALUES_SEPARATOR = ", ";
+    private static final String DEFAULT_STATEMENT_PARAMETER = "?";
+
+    public String generate(DatabaseTableMetadata tableMetadata) {
+        return generateInsertStatement(tableMetadata);
+    }
+
+    private String generateInsertStatement(DatabaseTableMetadata tableMetadata) {
```

Comment:
Why not inline this method in the one above and rename it to `generateInsertStatement`?